monkey/infection_monkey/dropper.py

```diff
@@ -43,18 +45,31 @@ def __init__(self, args):
         arg_parser.add_argument('-l', '--location')
         self.monkey_args = args[1:]
         self.opts, _ = arg_parser.parse_known_args(args)
-
+        self._default_server = None
+        self._default_server_port = None
         self._config = {'source_path': os.path.abspath(sys.argv[0]),
                         'destination_path': self.opts.location}
 
     def initialize(self):
         LOG.debug("Dropper is running with config:\n%s", pprint.pformat(self._config))
+        self._default_server = self.opts.server
```

Comment:
Whenever you copy paste the code your programmer's sense should be tingling and saying that maybe we can write a function/method that we could reuse rather than copy pasting the same code everywhere. Try to that here.